lib/std/os/uefi/protocols/device_path_protocol.zig

```diff
@@ -100,14 +106,26 @@ pub const DevicePathProtocol = packed struct {
 
             if (self.subtype == tag_val) {
                 // e.g. expr = .{ .Pci = @ptrCast(...) }
-                return @unionInit(TUnion, subtype.name, @ptrCast(subtype.field_type, self));
+                return @unionInit(TUnion, subtype.name, @ptrCast(
+                    subtype.field_type,
+                    @alignCast(@alignOf(subtype.field_type), self),
+                ));
```

Comment:
This alignCast shouldn't be here as its a pointer to (a subtype of) DevicePathProtocol which is unaligned.

Reply:
good catch, totally forgot about this